channeld/channeld.c

```diff
@@ -171,7 +171,7 @@ static void start_commit_timer(struct peer *peer);
 
 static void billboard_update(const struct peer *peer)
 {
-	const char *funding_status, *announce_status, *shutdown_status;
+	const char *funding_status = "", *announce_status = "", *shutdown_status = "";
```

Comment:
Compiler is wrong here but I suppose it cannot figure out that we have gotten all possible valid states, this is fine.